backend/requirements.in

```diff
@@ -1,3 +1 @@
-tfx==1.2.0
-kfp==1.7.1
```

Comment:
Note, pinning kfp SDK in backend was intentional, because backend is released at a different schedule than SDK.
If we remove the pin, backend will use the master branch's KFP SDK when it is released.
(when backend compiles sample pipelines)

Reply:
@zijianjoy you would still need to install `kfp` at least. Earlier, I was suggesting removing the pinned version, i.e. only the `==1.7.1` part. 
On a second thought, I think we can probably put `~=1.8` there, so it gets automatic patch and minor version upgrades, but not major upgrade.

Reply:
Sounds good and thank you both for the input! I added kfp~=1.8 here.